exasol_integration_test_docker_environment/lib/test_environment/ports.py

```diff
@@ -51,50 +44,46 @@ class PortsType(type):
 
     @property
     def default_ports(self) -> "Ports":
-        return Ports(database=8563, bucketfs=BucketFSPorts(), ssh=22)
+        return Ports(database=8563, bucketfs=2580, ssh=22, bucketfs_https=2581)
 
     @property
     def external(self) -> "Ports":
         # For external databases SSH port might depend on version database.
-        return Ports(database=8563, bucketfs=BucketFSPorts(), ssh=None)
+        return Ports(database=8563, bucketfs=2580, ssh=None, bucketfs_https=2581)
 
     @property
     def forward(self) -> "Ports":
-        return Ports(database=8563, bucketfs=BucketFSPorts(), ssh=20002)
+        return Ports(database=8563, bucketfs=2580, ssh=20002, bucketfs_https=2581)
 
 
 class Ports(metaclass=PortsType):
```

Comment:
If I understand this correctly, you decided against ports.bucketfs.https with the whacky idea of simulating an int. Is there are specific reason? I am not saying we should go for it, simply want to know what spoke against it.